src/rvwmo.adoc

```diff
@@ -76,22 +76,24 @@ simultaneously. All memory operations are single-copy atomic: they can
 never be observed in a partially complete state.
 (((operations, memory)))
 
-Among instructions in RV32GC and RV64GC, each aligned memory instruction
-gives rise to exactly one memory operation, with two exceptions. First,
-an unsuccessful SC instruction does not give rise to any memory
-operations.
-Second, floating-point load and store instructions of more than XLEN bits
-(e.g. FLD/FSD in RV32) may each give rise to multiple memory operations.
-An aligned AMO
-gives rise to a single memory operation that is both a load operation
-and a store operation simultaneously.
+Each aligned memory instruction that accesses XLEN or fewer bits gives rise
+to exactly one memory operation, unless specified otherwise. An aligned AMO
+gives rise to a single memory operation that is both a load operation and a
+store operation simultaneously.
 
 [NOTE]
 ====
-Future ISA
-extensions such as *V* (vector) and *P* (SIMD) may give rise to multiple
-memory operations. However, the memory model for these extensions has
-not yet been formalized.
+Among instructions in RV32GC and RV64GC, the following are exceptions to the
+rule that an aligned memory instruction gives rise to exactly one memory
+operation:
+
+* An unsuccessful SC instruction does not give rise to any memory operations.
+* Floating-point load and store instructions that access more than XLEN bits
+  (i.e., FLD/FSD in RV32) may each give rise to multiple memory operations.
```

Comment:
I'd either change this to "FLD/FSD in RV32 and FLQ/FSQ in RV64", or change the "i.e." back to "e.g.".  I guess I prefer just changing it to "e.g." and keeping it short.